x/oracle/types/genesis.go

```diff
@@ -10,28 +10,25 @@ import (
 func NewGenesisState(
 	params Params, rates []ExchangeRateTuple,
 	feederDelegations []FeederDelegation, penaltyCounters []PenaltyCounter,
-	aggregateExchangeRatePrevotes []AggregateExchangeRatePrevote,
 	aggregateExchangeRateVotes []AggregateExchangeRateVote,
 ) *GenesisState {
 	return &GenesisState{
-		Params:                        params,
-		ExchangeRates:                 rates,
-		FeederDelegations:             feederDelegations,
-		PenaltyCounters:               penaltyCounters,
-		AggregateExchangeRatePrevotes: aggregateExchangeRatePrevotes,
-		AggregateExchangeRateVotes:    aggregateExchangeRateVotes,
+		Params:                     params,
+		ExchangeRates:              rates,
+		FeederDelegations:          feederDelegations,
+		PenaltyCounters:            penaltyCounters,
+		AggregateExchangeRateVotes: aggregateExchangeRateVotes,
 	}
 }
 
 // DefaultGenesisState - default GenesisState used by columbus-2
 func DefaultGenesisState() *GenesisState {
 	return &GenesisState{
-		Params:                        DefaultParams(),
-		ExchangeRates:                 []ExchangeRateTuple{},
-		FeederDelegations:             []FeederDelegation{},
-		PenaltyCounters:               []PenaltyCounter{},
-		AggregateExchangeRatePrevotes: []AggregateExchangeRatePrevote{},
```

Comment:
do we need to remove this from the state in the upgrade handler?

Reply:
yes good point, I'll make that change